geopandas/geodataframe.py

```diff
@@ -1451,6 +1451,11 @@ def __setitem__(self, key, value):
     #
     # Implement pandas methods
     #
+    def copy(self, deep=True):
```

Comment:
```suggestion
    @doc(pd.DataFrame)
    def copy(self, deep=True):
```

to inherit the docstring